browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2546,6 +2546,9 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_CRYPTO_WALLET_ERROR_INSUFFICIENT_LIQUIDITY" desc="Error on crypto wallets">
       Insufficient liquidity
     </message>
+    <message name="IDS_CRYPTO_WALLET_ERROR_WRONG_NUMBER_FORMAT" desc="Error on crypto wallets">
+      Unrecognized number format
```

Comment:
@wchen342 could you please include screenshot here when that string is on the screen? @rmcfadden3 could you please check does that string make sense in that place?

Reply:
Screenshot:
![Screenshot](https://user-images.githubusercontent.com/6406743/231521088-f4d60dc6-3970-4263-9137-6449c5b94ca8.png)

Reply:
@SergeyZhukovsky Actually I just realized another problem, in EU locales the internal Java parser will parse `0.001` as 1 and in US locale it will parse `0,001` as 1 because dot is used as thousands separator in EU and comma is thousands separator in US. Shall we address this somehow or shall we keep this default behavior?

Reply:
Let's address in that PR. Just don't allow dots in EU locales and don't allow commas in US locales

Reply:
I found another way to disabling the thousands separators completely: https://github.com/brave/brave-core/pull/18016/files#diff-30eae7637d039986bb60e7d56d19a411894f2bce4725def3d4dd0945ea5edc72R1723

Reply:
does it let you enter fractional numbers?

Reply:
You mean decimals? Yes it can do decimals.

Reply:
Hey @SergeyZhukovsky & @wchen342 — Yeah, `Unrecognized number format` seems okay to me.

Reply:
thanks @rmcfadden3 , @wchen342 I mean 0.01 for the US locale and 0,01 for the EU.

Reply:
Yes these numbers work.